chibuezeayogu/React-Redux
# #161155249 Implement delete course

**What does the PR do?**
- This PR Implements  `Delete Course`

**Description of Task to be completed?**
- create delete course actions
- update course reducer to include course deletion
- add delete course modal
- write test
 
**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Redux.git)
-  cd into `React-Redux` and checkout to this branch `ft-delete-course-161155249`
- run the command `yarn dev` to start the application locally.
- run the command `yarn test` to ensure all tests are passing.

**What are the relevant pivotal tracker stories?**
- [#161155249](https://www.pivotaltracker.com/story/show/161155249)

**Screenshots**
- N/A
